static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -260,7 +260,8 @@ $quote-mark: 35px;
     }
 
     .content__article-body {
-        line-height: 1.4;
+        font-size: 17px;
```

Comment:
Ah, css originally came from: https://github.com/guardian/frontend/compare/line-height

Change was proposed by @blongden73 to see if it would help Spacefinder. We wanted to AB test the change to quantify any impact. Change does help so if Design would like to go ahead with this, or a similar change to the site, it works for Commercial! 😄